types/webpack/index.d.ts

```diff
@@ -849,7 +849,7 @@ declare namespace webpack {
     }
 
     class NamedChunksPlugin extends Plugin {
-        constructor(nameResolver: any);
+        constructor(nameResolver: (chunk: {}) => string | null );
```

Comment:
I think `nameResolver` should be optional.
BTW. If you make `chunk` typed as `{}`. It may throw errors when trying to get properties like `chunk.name` in strict mode. Consider using `any` or a much more accurate type.;